de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/ui/util/JobWithStatus.java

```diff
@@ -1,11 +1,11 @@
 package de.fu_berlin.inf.dpp.intellij.ui.util;
 
-import de.fu_berlin.inf.dpp.negotiation.ProjectNegotiation;
+import de.fu_berlin.inf.dpp.negotiation.ReferencePointNegotiation;
 
 /**
  * Helper class to run Runnables that return a
  */
 public abstract class JobWithStatus implements Runnable {
 
-    public ProjectNegotiation.Status status;
+    public ReferencePointNegotiation.Status status;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make status a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWcmJq28kEeD8QRkTSul&open=AWcmJq28kEeD8QRkTSul)

[ISSUE_KEY]: AWcmJq28kEeD8QRkTSul